committee-technical-oversight/sig-creation.md

```diff
@@ -0,0 +1,11 @@
+# SIG Creation
+
+Manual steps involved in creating a SIG.
+
+- Setup Mailing Lists
+  - `sig-{name}@spinnaker.io`
```

Comment:
Mention that `sig-{name}-leads` should be added as member of the group or is that too obvious to point out?